packages/drivers/local-driver/src/localResolver.ts

```diff
@@ -13,7 +13,7 @@ import {
     DriverHeader,
 } from "@fluidframework/driver-definitions";
 import { ScopeType } from "@fluidframework/protocol-definitions";
-import { generateToken } from "@fluidframework/server-services-client";
+import { generateToken } from "./auth";
 
```

Comment:
Is this api going away from this package?

Reply:
generateToken is already gone from server-services-client. It isn't isomorphic (doesn't work in both browser and server)